traptor/traptor.py

```diff
@@ -537,7 +565,7 @@ def main(sentry, stdout, info, debug, delay, id, type, key):
         if sentry:
             client = Client(SENTRY_SECRET)
             client.captureException()
-        logger.errror(e)
+        logger.error(e)
```

Comment:
Does this logger helper handle exception messages containing non-ascii characters? Or does that situation never arise?